index.html

```diff
@@ -5,15 +5,91 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <script src="https://kit.fontawesome.com/c5012398d3.js" crossorigin="anonymous"></script>
+    
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" type="text/css" media="screen and (min-device-width: 800px)" href="css/over800.css" />
```

Comment:
Using different stylesheets with media queries is perfectly valid but you need to take care that only one sheet is being applied at a time - in this case when you are on a wider device both will be active (I think you have noticed this based on comments). This is true of media queries in general but tends to cause more of a problem with this structure as it affects the whole stylesheet rather than a few selectors